src/get.sh

```diff
@@ -143,7 +150,7 @@ else
     exit 1
 fi
 
-if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]]; then
+if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]] && [[ ! version_greater_equal $KEPTN_VERSION 0.9.0 ]] ; then
```

Comment:
Hi!
As I'm the one responsible for introducing the original condition here, I'd like to weigh in.
my 2 cents: This was introduced when we were lacking darwin-arm64 support (we only had darwin-amd64). Keptn [0.10.0](https://github.com/keptn/keptn/releases/tag/0.10.0) and newer releases already contain a Keptn CLI binary for darwin-arm64".
If for whatever reason the user tries to download an older release on an M1 Mac, they will run into an error in the lines below
```
curl --fail -L "${URL}" --output ${FILENAME}
curl_exit_status=$?
if [ $curl_exit_status -ne 0 ]; then
    echo "An error occured while trying to download keptn from GitHub. Please manually download a release from https://github.com/keptn/keptn/releases."
    exit $curl_exit_status
fi
```

TL;DR: In the spirit of lowering the maintenance bill, we should get rid of 
```
if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]]; then
    echo "!!! Apple Silicon detected"
    echo "!!! Native Apple Silicon support is currently pending, falling back to amd64 for now."
    echo "!!! You should be able to run the Keptn cli using Rosetta 2 - read https://support.apple.com/en-us/HT211861 for more information)."
    KEPTN_ARCH="amd64"
fi
```

Reply:
Add: The initial backwards-compatibility check for Keptn versions below 0.8.0 was also removed over a year ago: https://github.com/keptn/get.keptn.sh/pull/53

Reply:
Hi @christian-kreuzberger-dtx  yes I realised we do not cover any version lower then 0.12 anyway so you are right I got rid of it :)